src/engraving/rw/write/twrite.cpp

```diff
@@ -1788,17 +1788,20 @@ void TWrite::write(const KeySig* item, XmlWriter& xml, WriteContext& ctx)
     writeItemProperties(item, xml, ctx);
     if (item->isAtonal()) {
         xml.tag("custom", 1);
-    } else if (item->isCustom()) {
-        xml.tag("accidental", int(item->key()));
-        xml.tag("custom", 1);
-        for (const CustDef& cd : item->customKeyDefs()) {
-            xml.startElement("CustDef");
-            xml.tag("sym", SymNames::nameForSymId(cd.sym));
-            xml.tag("def", { { "degree", cd.degree }, { "xAlt", cd.xAlt }, { "octAlt", cd.octAlt } });
-            xml.endElement();
-        }
     } else {
-        xml.tag("accidental", int(item->key()));
+        xml.tag("c_accidental", int(item->concertKey()));
```

Comment:
These properties should also be renamed to concertKey and actualKey respectively